src/collision/collider/constructor.rs

```diff
@@ -105,9 +105,9 @@ use bevy::utils::HashMap;
 /// }
 /// ```
 #[derive(Component, Clone, Debug, Default, PartialEq, Reflect)]
-#[reflect(Debug, Component, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serialize", reflect(Serialize, Deserialize))]
+#[reflect(Component, Debug, PartialEq)]
```

Comment:
```suggestion
#[reflect(Component, Debug, PartialEq, Default)]
```
while we're on it :)